test/ReCaptchaTest.php

```diff
@@ -20,18 +21,25 @@ class ReCaptchaTest extends TestCase
 
     public function setUp()
     {
-        $this->publicKey = getenv('TESTS_ZEND_SERVICE_RECAPTCHA_PUBLIC_KEY');
-        $this->privateKey = getenv('TESTS_ZEND_SERVICE_RECAPTCHA_PRIVATE_KEY');
-
-        if (empty($this->publicKey)
-            || $this->publicKey == 'public key'
-            || empty($this->privateKey)
-            || $this->privateKey == 'private key'
-        ) {
-            $this->markTestSkipped('ZendService\ReCaptcha\ReCaptcha tests skipped due to missing keys');
+        $this->siteKey = getenv('TESTS_ZEND_SERVICE_RECAPTCHA_SITE_KEY');
+        $this->secretKey = getenv('TESTS_ZEND_SERVICE_RECAPTCHA_SECRET_KEY');
+
+        if (empty($this->siteKey)) {
+            $this->siteKey = '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI';
```

Comment:
Maybe these should be in the `phpunit.xml.dist` as the default values, with a comment above indicating users should change them if they want to use their own keys? That would simplify the test code a bit.